pages/docs/reference/experimental.md

```diff
@@ -140,6 +140,18 @@ tasks.withType<KotlinCompile>().all {
 }
 ```
 
+If your Gradle module is multiplatform module, you have to use `useExperimentalAnnotation` method:
+
+```groovy
+sourceSets {
```

Comment:
Please add a multilanguage sample (as in the example above) with all `<div>`s and language names. 
In Kotlin DSL, use the `"kotlin.Experimental"` instead of `'kotlin.Experimental'` (double quotation marks instead of single).